advanced/Scripts/webpage.sh

```diff
@@ -346,7 +346,9 @@ CustomizeAdLists() {
 	elif [[ "${args[2]}" == "disable" ]]; then
 		sed -i "\\@${args[3]}@s/^http/#http/g" "${list}"
 	elif [[ "${args[2]}" == "add" ]]; then
-		echo "${args[3]}" >> ${list}
+		if [[ $(grep -c "${args[3]}" "${list}") -eq 0 ]] ; then
```

Comment:
You should probably be doing an *exact* comparison here to avoid the situation I described above from happening. Just use the proper anchors here:
```bash
grep -c "^${args[3]}$" "${list}"
```